README.md

```diff
@@ -22,148 +19,183 @@ Table of contents
-2. Update `ep-commerce/extensions/database/ext-data/src/main/resources/data/liquibase-changelog.xml`.<br/>
-Add the following sample data to the file. **Note:** It should be the only sample data included within the sample data blocks: `<include file="ep-blueprint-data/liquibase-changelog.xml" relativeToChangelogFile="true" />`
-3. Update `ep-commerce/extensions/database/ext-data/src/main/resources/environments/local/data-population.properties`. Set the liquibase.contexts property to `default,ep-blueprint`.
+* [Java JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html)
+* [Maven 3.5.2](https://archive.apache.org/dist/maven/maven-3/3.5.2/binaries/)
+* [IntelliJ IDEA](https://www.jetbrains.com/idea/) (optional)
+* A valid Elastic Path development environment. For  more information, see
+[The Starting Construction Guide](https://developers.elasticpath.com/commerce/construction-home).
+
+### Configuration Parameter Descriptions
+
+Configure the following parameters in the `./src/ep.config.json` file:
+
+|  Parameter| Importance|Type|Description|
+|--|--|--|--|
+|`cortexApi.path`|Required|String|The URL, which is composed of the hostname and port, to access Cortex. By default, a web proxy is configured in the [Webpack](https://webpack.js.org/) configuration of this project. For local development, set this value to `/cortex` to redirect Cortex calls to the local proxy.|
+|`cortexApi.scope`|Required|String|Name of the store from which Cortex retrieves data.|
+|`cortexApi.pathForProxy`|Required|String|The path to which the [Webpack](https://webpack.js.org/) proxy routes the Cortex calls from the storefront. This value is a URL that consists of hostname and port of a running instance of Cortex. Leave this field blank to disable proxy.|
+|`skuImagesUrl`|Required|String|The URL that consists of the path to catalog images hosted on an external CMS. Set this parameter to the complete URL of the images by replacing the `sku/file-name` with the `%sku%` string . This value is populated when the page is loaded with values retrieved by Cortex.|
+|`siteImagesUrl`|Optional|String|Path to the site content and marketing images hosted on an external CMS. Set this parameter to the complete URL of the images by replacing the filename and file extension with the `%fileName%` . This parameter is populated with the values set in the components when the page is loaded, and use the assets locally available in the `./src/images/site-images` directory.|
+|`enableOfflineMode`|Optional|String|The option to enable offline mode to fetch requests static data instead of Cortex. For more information, see [offline mode](#offline_mode).|
+|`gaTrackingId`|Optional|String|
+ The Google Analytics tracking ID to integrate with Google Analytics Suite to track enhanced e-commerce activity on the site.|
```

Comment:
The formatting here is incorrect and causes the text to wrap to a new row in the table.